src/MigrationTools.Clients.AzureDevops.Rest/Endpoints/AzureDevOpsEndpoint.cs

```diff
@@ -182,7 +182,7 @@ public async Task<IEnumerable<DefinitionType>> GetApiDefinitionsAsync<Definition
                         }
                         else
                         {
-                            Log.LogError($"Failed on call to get single [{typeof(DefinitionType).Name}] with Id [{definition.Id}].\r\nUrl: GET {response.RequestMessage.RequestUri.ToString()}");
+                            Log.LogError($"Failed on call to get single [{typeof(DefinitionType).Name}] with Id [{definition.Id}].\r\nUrl: GET {response.RequestMessage.RequestUri.ToString()}\r\nResponse Code:{httpResponse.StatusCode}");
```

Comment:
This way of running the error will not result in good logs of aubmition to AI.

Please use Log.LogError("messagewith {value} in it", value)"

Reply:
Like this update?